polyfill/spatial-navigation-polyfill.js

```diff
@@ -167,14 +167,17 @@
       if (eventTarget.nodeName === 'IFRAME')
         eventTarget = eventTarget.contentDocument.body;
 
+      let bestInsideCandidate = null;
+
       // 5-2
       if (getCSSSpatNavAction(eventTarget) === 'scroll') {
         if (scrollingController(eventTarget, dir)) return;
       } else if (getCSSSpatNavAction(eventTarget) === 'focus') {
-        if (focusingController(eventTarget.spatialNavigationSearch(dir, {candidates: getSpatialNavigationCandidates(eventTarget, {mode: 'all'}), inside: true}), dir)) return;
+        bestInsideCandidate = eventTarget.spatialNavigationSearch(dir, {container: eventTarget, candidates: getSpatialNavigationCandidates(eventTarget, {mode: 'all'}), inside: true});
+        if (focusingController(bestInsideCandidate, dir)) return;
       } else if (getCSSSpatNavAction(eventTarget) === 'auto') {
-        if (focusingController(eventTarget.spatialNavigationSearch(dir, {inside: true}), dir)) return;
-        if (scrollingController(eventTarget, dir)) return;
+        bestInsideCandidate = eventTarget.spatialNavigationSearch(dir, {container: eventTarget, inside: true});
+        if (focusingController(bestInsideCandidate, dir) || scrollingController(eventTarget, dir)) return;
```

Comment:
When I read this code, changed logic is only one part ("container: eventTarget,")
is it right?
is there any reason to change other part?
I think bestInsideCandidate is not needed. because it is one-time used variable and the readability become worse.

Reply:
Not only that part. 
If without the variable, the second condition (the case of `spatial-navigation-action: focus`) will be

```js

else if (getCSSSpatNavAction(eventTarget) === 'focus') {
        if (focusingController(eventTarget.spatialNavigationSearch(dir, {container: eventTarget, candidates: getSpatialNavigationCandidates(eventTarget, {mode: 'all'}), inside: true}), dir)) return;

```

And I think this decreases the readability of the code.

Reply:
I mean 
> **changed logic** is only one part ("container: eventTarget,")
and, creating new variable is not logical change.

there is two reason such change should be avoid.
1. Such change make me difficult to figure out difference.
2. One time variable should be avoided. (memory, code managing)

Reply:
I got your point! I'll do refactoring after fixing the bugs.